### PR TITLE
Switch to issue forms

### PR DESCRIPTION
**Pull request by Andre601**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
[pull-request]: https://github.com/JDA-Applications/JDA-Utilities/pulls

#### Pull Request Checklist
Please follow the following steps before opening this PR.<br>
PRs that do not complete the checklist will be subject to denial for
missing information.

- [x] I have checked the [pull request page][pull-request] for upcoming
      or merged features/bug fixes.
- [x] I have read JDA's [contributing guidelines][contributing].

#### Pull Request Information
Check and fill in the blanks for all that apply:

- [ ] My PR fixes a bug, error, or other issue with the library's codebase.
- [ ] My PR is for the `______` module of the JDA-Utilities library.
- [ ] My PR creates a new module for the JDA-Utilities library: `______`.
- [x] Other: Switching from issue templates to the new issue forms.

#### Description
The repo still uses the original issue template which not only is kinda stupid in its general format but also links to the wrong places.

I've gone ahead and switched the issue template with two issue forms instead. One for Bug reports and one for Feature request.
I also created a config.yml for the template selector to link to places such as the wiki and Discord server.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- type: dropdown
+  attributes:
+    label: Affected Modules
+    description: Which modules do you have issues with? Multiple selections possible.
+    multiple: true
+    options:
+    - Command
+    - Commons
+    - Menu
+    - OAuth2
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/bug_report.yml:41]: Should examples or doc be added here?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Description
+    description: |-
+      Provide a detailed explanation about your Bug report including any errors.
+      Please make sure that errors and code-snippets are either shared through code blocks or sites such as https://hastebin.de
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
      Please make sure that errors and code-snippets are either shared through code blocks or sites such as https://paste.gg
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    about: "Join the Discord Server for quick support."
+  - name: Wiki
+    url: https://github.com/Chew/JDA-Chewtils/wiki
+    about: "Check the JDA-Chewtils wiki for useful information."
+  - name: Migrate from JDA-Utilities
+    url: https://github.com/Chew/JDA-Chewtils/wiki/Migrating-from-JDA-Utilities
+    about: "Want to switch over from JDA-Utilities? This page has you covered!"
+  - name: Releases
+    url: https://github.com/Chew/JDA-Chewtils/releases
+    about: "The latest releases of JDA-Chewtils can be found here."
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/config.yml:14]: Link to the discussions maybe?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      Thank you for opening a Feature request!
+      Please note that the issue tracker is reserved for issues, errors and feature requests regarding JDA-Chewtils.
+      
+      **Want to make a Bug report instead?**
+      Use the [Bug Report](https://github.com/Chew/JDA-Chewtils/issues/new?template=bug_report.yml) template!
+      
+      - For examples, check the [examples module](https://github.com/Chew/JDA-Chewtils/tree/master/examples).
+      - For questions join the [Discord Server of Chew](https://discord.gg/SUGVxJpg8r) and ask in the `#jda-chewtils` channel.  
+        **Do not contact the JDA-Team for questions regarding JDA-Chewtils!**
+      - For general programming questions ask on [StackOwnerflow](https://stackoverflow.com/questions/tagged/java) instead.
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/feature_request.yml]:
```suggestion
      - For general programming questions ask on [StackOverflow](https://stackoverflow.com/questions/tagged/java) instead.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      Thank you for opening a Bug report!
+      Please note that the issue tracker is reserved for issues, errors and feature requests.
+      
+      **Want to make a Feature request instead?**
+      Use the [Feature request](https://github.com/Chew/JDA-Chewtils/issues/new?template=feature_request.yml) template!
+      
+      - For examples, check the [examples module](https://github.com/Chew/JDA-Chewtils/tree/master/examples).
+      - For questions join the [Discord Server of Chew](https://discord.gg/SUGVxJpg8r) and ask in the `#jda-chewtils` channel.  
+        **Do not contact the JDA-Team for questions regarding JDA-Chewtils!**
+      - For general programming questions ask on [StackOwnerflow](https://stackoverflow.com/questions/tagged/java) instead.
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
      - For general programming questions ask on [StackOverflow](https://stackoverflow.com/questions/tagged/java) instead.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- type: dropdown
+  attributes:
+    label: Affected Modules
+    description: Which modules is this Feature request for? Multiple selections possible.
+    multiple: true
+    options:
+    - Command
+    - Commons
+    - Menu
+    - OAuth2
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/feature_request.yml:41]: see above

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Description
+    description: |-
+      Provide a detailed explanation about your Feature request including any code-examples and use-cases.
+      Please make sure that code-snippets are either shared through code blocks or sites such as https://hastebin.de
```
Comment (Chew) [inline .github/ISSUE_TEMPLATE/feature_request.yml]:
```suggestion
      Please make sure that code-snippets are either shared through code blocks or sites such as https://paste.gg
```

**Review by Chew — COMMENTED** [review]
god I hate the old templates so much, thank you for doing this for me 😅 
just a few suggestions and discussion points

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/bug_report.yml:41]: I think I can add the Examples one, but docs isn't really a module, no? More the folder of the generated Javadocs, right?

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/config.yml:14]: Will add.

**Comment by Andre601** [comment]
Any more changes you like?

**Review by Chew — APPROVED** [review]
Great work!

**Pull request merged**